### PR TITLE
Sml workload vpc

### PR DESCRIPTION
**Pull request by rashidatsalau**
Kindly review my code. the ticket number is 03. thank you.

### CODE REVIEW

File: migration-lab/terraform/migration_vpc/vpc_peering.tf [hunk 1]
```diff
@@ -0,0 +1,9 @@
+
+resource "aws_vpc_peering_connection" "vpc_peering_connection" {
+  peer_vpc_id   = aws_vpc.vpc.id
+  vpc_id        = aws_vpc.mgn_vpc.id
+  
+  tags = {
+    Name = "VPC Peering between corporate_data_centre and migrated_resouces"
+  }
+} #in the peer_vpc_id, it needs to correspond to the on-prem.
```
Comment (olga-matusik) [inline migration-lab/terraform/migration_vpc/vpc_peering.tf:9]: I saw in the terraform documentation (https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/vpc_peering_connection) that we can use `auto_accept = true` if both VPCs are in the same account and region.

**Review by olga-matusik — CHANGES_REQUESTED** [review]

**Review by olga-matusik — CHANGES_REQUESTED** [review]
According to the ticket we need also some NACL rules here.

File: migration-lab/terraform/migration_vpc/NACL.tf [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+    action     = "deny"
+    cidr_block = "10.0.0.0/16"
+    from_port  = 0
+    to_port    = 0
+  }
+   
+
+  egress {
+    protocol   = "-1"
+    rule_no    = 200
```
Comment (olga-matusik) [inline migration-lab/terraform/migration_vpc/NACL.tf:20]: rule_no here can also be 100

File: migration-lab/terraform/migration_vpc/NACL.tf [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+    rule_no    = 100
+    action     = "deny"
+    cidr_block = "10.0.0.0/16"
+    from_port  = 0
+    to_port    = 0
+  }
+   
+
+  egress {
+    protocol   = "-1"
```
Comment (olga-matusik) [inline migration-lab/terraform/migration_vpc/NACL.tf:19]: I think you don't need " ", just -1 is enough

File: migration-lab/terraform/migration_vpc/NACL.tf [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+    action     = "deny"
+    cidr_block = "10.0.0.0/16"
+    from_port  = 0
+    to_port    = 0
+  }
+
+  tags = {
+    Name = "NACL"
+  }
+}
```
Comment (olga-matusik) [inline migration-lab/terraform/migration_vpc/NACL.tf:30]: I think we need to allow also the traffic from staging area subnet, I checked it on my account, and without that all traffic is denied...

**Review by olga-matusik — CHANGES_REQUESTED** [review]

**Review by olga-matusik — APPROVED** [review]
LGTM! 👍

**Review by zeynepiskender — APPROVED** [review]

**Pull request merged**